insights/parsers/init_process_cgroup.py

```diff
@@ -0,0 +1,55 @@
+"""
+InitProcessCgroup - File ``/proc/1/cgroup``
+===========================================
+
+This parser reads the content of ``/proc/1/cgroup``.
+This file shows the cgroup detail of init process.
+The format of the content is like key-value. We can
+also use this info to check if the archive is from
+container or host.
+"""
+
+from .. import parser, CommandParser, LegacyItemAccess
+from insights.specs import Specs
+
+
+@parser(Specs.init_process_cgroup)
+class InitProcessCgroup(CommandParser, LegacyItemAccess):
+    """
+    Class ``InitProcessCgroup`` parses the content of the ``/proc/1/cgroup``.
+
+    Attributes:
+        is_container (bool): It is used to check if a archive is from host or container.
+        Return True if the archive is from container.
```

Comment:
@wushiqinlou you need to indent this line to associate it with the preceding line in the docs.  You can review the generated docs to see the problem.  Otherwise this PR looks good.